### PR TITLE
EAP7-1156: Add test for open ports

### PR DESCRIPTION
**Pull request by hpehl**
Upstream Issue: https://issues.jboss.org/browse/HAL-532
EAP7 Issue: https://issues.jboss.org/browse/EAP7-1156

### CODE REVIEW

File: tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/OpenPortsTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import static org.junit.Assert.assertFalse;
+
+@RunWith(Arquillian.class)
+@Category(Standalone.class)
+public class OpenPortsTest {
+
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final ServerEnvironmentUtils serverEnvironmentUtils = new ServerEnvironmentUtils(client);
+    private static final Operations operations = new Operations(client);
+
```
Comment (spriadka) [inline tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/OpenPortsTest.java]: Please close the `client` after the tests e.g by calling `client.close()` in the `@AfterClass` method

File: tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/microprofile/health/MicroProfileHealthCheckTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import static org.jboss.hal.testsuite.fragment.finder.FinderFragment.runtimeSubsystemPath;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(Arquillian.class)
+@Category(Standalone.class)
+public class MicroProfileHealthCheckTest {
+
+    private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
+    private static final ServerEnvironmentUtils serverEnvironmentUtils = new ServerEnvironmentUtils(client);
+
```
Comment (spriadka) [inline tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/microprofile/health/MicroProfileHealthCheckTest.java]: As mentioned in `OpenPortsTest` close the client after test execution

File: tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/OpenPortsTest.java (hunk 1 above)
Comment (hpehl) [reply tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/OpenPortsTest.java]: Added code to close client in `@AfterClass` method.

File: tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/microprofile/health/MicroProfileHealthCheckTest.java (hunk 2 above)
Comment (hpehl) [reply tests/basic/src/test/java/org/jboss/hal/testsuite/test/runtime/microprofile/health/MicroProfileHealthCheckTest.java]: Added code to close client in `@AfterClass` method.

**Comment by hpehl** [comment]
@spriadka should be ready to merge now.

**Comment by spriadka** [comment]
Thanks, @hpehl, have you rebased somehow the branch? Please reduce the amount of commits and changed files

**Comment by hpehl** [comment]
Rebase seems to cause no conflicts. Merging PR.

**Pull request merged**